src/components/LibrariesList.tsx

```diff
@@ -56,9 +56,8 @@ export class LibrariesList extends React.Component<LibrariesListProps, Libraries
         }
         </ul>
       );
-    }
-    else {
-      return <span className="page-header">There are no libraries in this registry yet.</span>;
+    } else {
+        return <span className="page-header">There are no libraries in this registry yet.</span>;
```

Comment:
This should still stay at the 2 spaces indentation though.

Reply:
That looks SO WRONG to me.  Weird.